ui/src/testing/stubs/cohort-builder-service-stub.ts

```diff
@@ -403,9 +401,7 @@ export const SurveyQuestionStubVariables = {
 
 export class CohortBuilderServiceStub extends CohortBuilderApi {
   constructor() {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.